docs/development/digital_control.md

```diff
@@ -1 +1,127 @@
+            alt = "analog_controller"
+            height = auto
+            width = "500px" />
+        <figcaption>Fig.2 Analog Controller</figcaption>
+    </center>
+</figure>  
+
+
+In Analog controllers, UVLO comparator is used whose input is strictly analog and the output is digital signal i.e. either high (1) or low (0).
+Depending upon this value the analog PWM increases or decreases the duty cycle of the power stage. But the output of the comparator carries limited information of parameter (i.e whether the monitored value is above or below the threshold value) and does not carry any information regarding the amount of the deviation from the threshold which results in less precise controlling of the parameter.
+
+
+### Digital Controller:
+
+Figure 2 shows generic digital controller that provides voltage regulation.
+
+<figure>
+    <center>
+        <img 
+            src = "./images/digital_controller.svg" 
+            alt = "digital_controller"
+            height = auto
+            width = "500px"/>
+        <figcaption>Fig.2 Digital Controller</figcaption>
+    </center>
+</figure>
+
+
+This type of controller uses Standard communication block, Processing unit, power management, memory, I/O's and general purpose ADCs. These controllers are not only capable of doing regulation but also can perform complex algorithms to monitor and maintain few parameters within permissible limits.
+
+The three main blocks of this controller are ADC which samples the error voltage with respect to reference voltage, Digital filter which is used to compensate the error and Digital Pulse width modulator which converts compensated signal into Transistor gate drivable signals.
+
+The advantage of the digital controller is the flexibility to decide the right course of action to a given stimuli but at the same time these decisions have to be hardcoded(software) by designer. However, in case of analog controllers, these decisions were made available by the semiconductor manufacturers.
+
+## Modelling the Digital controller 
+In order to design controller for any system, it is necessary to have knowledge about system behaviour. This is nothing but mathematical description of the relation among inputs to the process state variables, and output. When we add any digital components into the system, we need consider few effects like feedback quantisation, Control effort quantization, delay involved in sampling the feedback signal and time to calculate the control effort.
+
+Figure 3 shows closed loop block diagram of the digitally controlled DC-DC converter.   
+
+<figure>
+    <center>
+        <img 
+            src = "./images/system_model.svg" 
+            alt = "system_model"
+            height = auto
+            width = "700px" />
+        <figcaption>Fig.3 Closed loop system</figcaption>
+    </center>
+</figure>  
+
+The closed-loop gain, from the PWM control effort (u) to the sensed output voltage (V<sub>sense</sub>) is given by
```

Comment:
I suggest to use in-line latex equation here ($V_{sense}$ to have the same font as in below equations.

Reply:
ya sure